packages/client/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @lens-protocol/client
 
+## 1.0.0-beta.1
+
+### Minor Changes
+
+- 7af4585: Removed default export of main client class, added LensClient as named export
```

Comment:
this is a major change, but not sure how to put it in the changelog

Reply:
sorted for now